ResearchKit/Common/ORKOrderedTask.h

```diff
@@ -427,13 +416,6 @@ ORK_EXTERN ORKTrailMakingTypeIdentifier const ORKTrailMakingTypeIdentifierB;
                                                       duration:(NSTimeInterval)duration
                                                    handOptions:(ORKPredefinedTaskHandOption)handOptions
                                                        options:(ORKPredefinedTaskOption)options;
-/**
- @Deprecated
- */
-+ (ORKOrderedTask *)twoFingerTappingIntervalTaskWithIdentifier:(NSString *)identifier
-                                        intendedUseDescription:(nullable NSString *)intendedUseDescription
-                                                      duration:(NSTimeInterval)duration
-                                                       options:(ORKPredefinedTaskOption)options __deprecated;
```

Comment:
This is handled in https://github.com/ResearchKit/ResearchKit/pull/959 and will cause a merge conflict with that branch.